salesagility/SuiteCRM
# Fix/events calendar labels

## Description



This fix is to label the fields within the events pop up from the calendar dashlet.

## Motivation and Context

This specific popup had a number of missing labels causing the values to seem ambiguous.

## How To Test This

1) Create an event within the events module
2) Navigate to the home page
3) Ensure that the calendar dashlet is visible
4) Hover over the newly created event
5) The fields should now have appropriate labels in the popup box
6) Also, click the body of the event within the dashlet
7) View the larger popup and ensure that the appropriate labels are visible, e.g. all buttons, fields and title

## Types of changes


### Final checklist
